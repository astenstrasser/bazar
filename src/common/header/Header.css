.header {
 display: flex;
 justify-content: center;
 margin-top: 5rem;
}

.header-logo{
 width: 30rem;
 max-width: 100%;
}
