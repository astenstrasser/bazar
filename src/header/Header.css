.header {
 display: flex;
 justify-content: center;
 margin-top: 5rem;
}

.header-logo{
 width: 30rem;
}
