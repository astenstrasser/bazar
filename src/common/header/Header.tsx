import React from "react";
import "./Header.css";
import Logo from "../../assets/logo.png";

const Header: React.FC = () => {
  return (
    <div className="header">
      <a href={"/"}>
        <img className="header-logo" src={Logo} alt={"logo bazar"} />
      </a>
    </div>
  );
};

export default Header;
