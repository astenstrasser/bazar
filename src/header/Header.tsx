import React from "react";
import "./Header.css";
import Logo from "../assets/logo.png";

const Header: React.FC = () => {
  return (
    <div className="header">
      <img className="header-logo" src={Logo} alt={"logo bazar"} />
    </div>
  );
};

export default Header;
