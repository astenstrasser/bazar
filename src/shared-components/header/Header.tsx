import React from "react";
import "./Header.css";
import Logo from "../../assets/logo.png";
import GoogleAuthentication from "../google-authentication/GoogleAuthentication";

const Header: React.FC = () => {
  return (
    <div className="header">
      <a href={"/"}>
        <img className="header-logo" src={Logo} alt={"logo bazar"} />
      </a>
      <div className="google-login">
        <GoogleAuthentication />
      </div>
    </div>
  );
};

export default Header;
