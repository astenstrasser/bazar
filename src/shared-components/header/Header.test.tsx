import React from "react";
import Header from "./Header";
import { render } from "@testing-library/react";

test("renders snapshot", () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});
