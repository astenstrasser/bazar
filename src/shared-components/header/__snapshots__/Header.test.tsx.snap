// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<div>
  <div
    class="header"
  >
    <a
      href="/"
    >
      <img
        alt="logo bazar"
        class="header-logo"
        src="logo.png"
      />
    </a>
  </div>
</div>
`;
